test/controllers/journeys/adjustments/profitOrLoss/CurrentYearLossControllerSpec.scala

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.journeys.adjustments.profitOrLoss
+
+import base.ControllerSpec
+import cats.implicits.catsSyntaxOptionId
+import controllers.standard.{routes => genRoutes}
+import forms.adjustments.profitOrLoss.WhatDoYouWantToDoWithLossFormProvider
+import models.NormalMode
+import models.common.UserType
+import models.database.UserAnswers
+import models.journeys.adjustments.WhatDoYouWantToDoWithLoss
+import models.journeys.adjustments.WhatDoYouWantToDoWithLoss.DeductFromOtherTypes
+import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
+import pages.adjustments.profitOrLoss.WhatDoYouWantToDoWithLossPage
+import play.api.Application
+import play.api.data.Form
+import play.api.http.Status.OK
+import play.api.i18n.Messages
+import play.api.mvc.Results.Redirect
+import play.api.mvc.{Call, Request}
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import stubs.services.SelfEmploymentServiceStub
+import views.html.journeys.adjustments.profitOrLoss.WhatDoYouWantToDoWithLossView
+
+class CurrentYearLossControllerSpec extends ControllerSpec {
+
+  def answer: Set[WhatDoYouWantToDoWithLoss] = Set(DeductFromOtherTypes)
```

Comment:
Is this not supposed to use the CheckboxControllerBaseSpec that you edited in this PR for this purpose?